OpenCore-Catalina/EFI/OC/Kexts/VirtualSMC.kext/Contents/Info.plist

```diff
@@ -177,7 +177,7 @@
 						gA==
 						</data>
 						<key>comment</key>
-						<string>This should be 1 on laptops, and is overriden by sensors</string>
+						<string>This should be 1 on laptops, and is overridden by sensors</string>
```

Comment:
Please report this to https://github.com/acidanthera/VirtualSMC (upstream project).